test/integration/ggrc/notifications/test_assessment_notifications.py

```diff
@@ -419,3 +421,32 @@ def test_import_snapshot_mapped(self, status):
     ]))
     self._check_csv_response(response, {})
     self.assert_asmnt_notifications()
+
+  def test_notification_create_time(self):
+    """Test for Assessment created date verification"""
+    audit = factories.AuditFactory()
+    time_stamp = datetime.now(timezone('US/Pacific'))
+    date_format = '%m/%d/%Y %H:%M:%S %Z'
+    converted_date = time_stamp.strftime(date_format)
+    assignee_acr = all_models.AccessControlRole.query.filter_by(
+        object_type="Assessment",
+        name="Assignees",).first()
+    self.api.post(Assessment, {
+        "assessment": {
+            "title": "Assessment1",
+            "context": None,
+            "audit": {
+                "id": audit.id,
+                "type": "Audit",
+            },
+            "access_control_list": [
+                acl_helper.get_acl_json(self.primary_role_id, self.auditor.id),
+                acl_helper.get_acl_json(assignee_acr.id, self.auditor.id)],
+            "status": "In Progress",
+        }
+    })
+    notifs, notif_data = common.get_daily_notifications()
+    created_date = notif_data["user@example.com"]["assessment_open"].values()
```

Comment:
It is clearly not a `created_date`. Please rename it to something like `asmts_data`.
```suggestion
    asmts_data = notif_data["user@example.com"]["assessment_open"].values()
```